#!/usr/bin/env python
PACKAGE = "final_pkg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

integration_method_enum = gen.enum([gen.const("Euler", int_t, 0, "euler integration method"), 
		gen.const("Runge_Kutta", int_t, 1, "Runge-Kutta integration method")], 
		"an enum which represents the integration method used to calculate odometry")

gen.add("integration_method", int_t, 0, "a parameter to set the integration method used to calculate odometry", 
		0, 0, 1, edit_method = integration_method_enum)

exit(gen.generate(PACKAGE, "pub_odometry", "parameters"))
